bedrock/redirects/redirects.py

```diff
@@ -2061,6 +2061,7 @@
              'https://www.thunderbird.net/thunderbird/{version}/releasenotes/'),
     redirect(r'^thunderbird/(?P<version>[^/]+)/system-requirements/$',
              'https://www.thunderbird.net/thunderbird/{version}/system-requirements/'),
+    redirect(r'^thunderbird', 'https://www.thunderbird.net/'),
```

Comment:
Instead of adding another redirect here I *think* you can just add a `?` after `thunderbird/` in the existing rule below?

```python
redirect(r'^thunderbird/?(?P<path>.*)', 'https://www.thunderbird.net/{path}'),
```